package.json

```diff
@@ -11,17 +11,18 @@
     "sw-runtime-caching": "0.0.17"
   },
   "devDependencies": {
+    "@11ty/eleventy": "^0.11.0",
     "chalk": "^2.4.1",
     "chalk-cli": "^3.0.0",
     "imagemin": "^5.3.1",
-    "imagemin-mozjpeg": "^7.0.0",
+    "imagemin-mozjpeg": "^9.0.0",
     "imagemin-pngquant": "^5.1.0",
     "inquirer": "^6.0.0",
     "node-fetch": "^2.1.2",
     "prettier": "^1.13.7",
     "purgecss": "^1.0.1",
     "request-x-ray": "^0.1.4",
-    "sharp": "^0.20.5",
+    "sharp": "^0.25.4",
```

Comment:
Can we create an issue to set up Renovate after we merge this stuff? I'm spoiled by the way it auto-pins and updates dependencies!

Reply:
https://github.com/cloudfour/pwastats/issues/212